xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp

```diff
@@ -174,6 +181,13 @@ void CVideoLayerBridgeDRMPRIME::Configure(CVideoBufferDRMPRIME* buffer)
     m_DRM->AddProperty(plane, "COLOR_RANGE", value);
 
   auto connector = m_DRM->GetConnector();
+
+  // set max bpc to allow the drm driver to choose a deep colour mode
+  int bpc = picture.colorBits > 8 ? 12 : 8;
+  result = m_DRM->AddProperty(connector, "max bpc", bpc);
```

Comment:
Is this guaranteed to be available? Should it be guarded with `SupportsProperty("max bpc")`?